molecule/default/prepare.yml

```
@@ -18,13 +18,14 @@
 - name: Wait for systemd to complete initialization (Fedora)
   hosts: os_Fedora
   tasks:
-    - name: Wait for systemd to complete initialization # noqa 303
+    - name: Wait for systemd to complete initialization # noqa command-instead-of-module
       ansible.builtin.command: systemctl is-system-running
       register: systemctl_status
       until: "'running' in systemctl_status.stdout"
       retries: 30
       delay: 5
-      when: ansible_service_mgr == 'systemd'
+      when: ansible_service_mgr == "systemd"
+      changed_when: "'running' in systemctl_status.stdout"
```

The command `systemctl is-system-running` doesn't actually change anything, so I think this line should instead be:
```suggestion
      changed_when: false
```